scripts/prettier/index.js

```diff
@@ -40,14 +40,14 @@ if (!files.length) {
   process.exit(0);
 }
 
-files.forEach(file => {
-  const options = prettier.resolveConfig.sync(file, {
+files.forEach(async file => {
```

Comment:
This requires `Promise.all` and `files.map` now, and `prettier.check()` further down needs to be awaited, otherwise the process will exit with `0` even if there are unformatted files (in `check` or `check-changed` mode).

Sorry if I'm being a nuisance here. You know, open source is strange. Sometimes it feels like you're watching an NBA game, and suddenly decide to run on the court, steal the ball from the point guard, and try to throw a lob pass to the center. But he doesn't catch it and everyone gives you a stare.

Reply:
Haha no need to apologize, good to have another pair of eyes